samples/net/openthread/coap_client/Kconfig

```diff
@@ -0,0 +1,29 @@
+mainmenu "logger_module_filtering"
+
+source "$ZEPHYR_BASE/Kconfig.zephyr"
+
+config COAP_CLIENT_MOD_LOG_LEVEL
```

Comment:
Zephyr has a template for this, see how it's done for instance [here](https://github.com/lmaciejonczyk/fw-nrfconnect-nrf/blob/master/applications/asset_tracker/Kconfig#L393).